google-cloud-tools-plugin/src/com/google/gct/idea/appengine/cloud/AppEngineCloudType.java

```diff
@@ -202,6 +203,16 @@ public void setName(String name) {
         this.name = name;
       }
 
+      @Nullable
+      @Override
+      public File getFile() {
+        return new File(userSpecifiedFilePath);
```

Comment:
As a side note, I think adding an error dialog into the mix would be a bit odd since we are already showing label error messages. So having the one off dialog may feel strange.